cost/scheduled_report_unallocated/scheduled_report_unallocated.pt

```diff
@@ -0,0 +1,391 @@
+name "Scheduled Report for Unallocated Costs"
+rs_pt_ver 20180301
+type "policy"
+short_description "This policy allows you to set up scheduled reports that will provide summaries of cloud cost that are unallocated for the dimensions you specify, delivered to any email addresses you specify."
+long_description ""
+severity "low"
+category "Cost"
+default_frequency "monthly"
+tenancy "single"
+info(
+  version: "0.1.0",
+  publish: "false",
+  provider: "Flexera Optima",
+  service: "",
+  policy_set: ""
+  )
+
```

Comment:
Possibly a nitpick, given that it/s an unpublished 0.1 version, but I think the sections should still have comments clearly labelling and separating them like so:

###############################################################################
# Datasources & Scripts
###############################################################################